packages/core/contracts/polygon/OracleBaseTunnel.sol

```diff
@@ -16,14 +15,25 @@ abstract contract OracleBaseTunnel {
         int256 price;
     }
 
+    // This value must be <= the Voting contract's `ancillaryBytesLimit` value otherwise it is possible
+    // that a price can be requested to this contract successfully, but cannot be resolved by the DVM which refuses
+    // to accept a price request made with ancillary data length over a certain size.
+    uint256 public constant ancillaryBytesLimit = 8192;
```

Comment:
- The `OracleChildTunnel` contract does not validate that price requests have [sufficiently short](https://github.com/UMAprotocol/protocol/blob/d146d4feddcf99e157a909032a3071e95ca51eb9/packages/core/contracts/oracle/implementation/Voting.sol#L266) ancillary data (after the [context is added](https://github.com/UMAprotocol/protocol/blob/d146d4feddcf99e157a909032a3071e95ca51eb9/packages/core/contracts/polygon/OracleChildTunnel.sol#L121)). This could lead to price requests that cannot be resolved. Consider validating the length of the ancillary data before initiating a price request.

@UMAprotocol/eng I decided to resolve this comment by adding a feature to `OracleBaseTunnel` instead of `OracleChildTunnel`. What do you think about addressing it here versus in the implementing `OracleChildTunnel`?

Reply:
Why not do it in the child contract? Child seems better since it will revert immediately rather than having to cross the bridge and _then_ revert.